package-managers/nuget.md

```diff
@@ -1,25 +1,76 @@
 # NuGet \(.NET\)
-
 TODO (inspiration: http://docs.myget.org/docs/walkthrough/getting-started-with-nuget)
 
-## Connecting to a NuGet feed
 
+## Connecting to a NuGet feed
 public feed, private feed with basic auth, pre-authenticated feed URL
 
-### Credential Provider for Visual Studio
 
+### Find your feed URL
+
+#### Public feed
+There are two types of public feeds you can connect to. A public feed you own and one from the gallery. The only real difference here, is where to find the URL of the feed.
+
+With this URL you can connect to the feed using your favorite tool, which could be [Visual Studio](#visual-studio), [JetBrains Rider](#jetbrains-rider) or the [NuGet-CLI](#nugetexe)
+
+
+##### Gallery feed
+When connecting to a gallery feed you first click on the feed you want to connect to and there click on the "Connect to feed" button.

```

Comment:
Suggestion: `After browsing to a gallery feed, we can get its connection details using the **Connect to feed** button. This will open a dialog where we can find the various endpoints for this feed.`